homeassistant/components/tradfri/config_flow.py

```diff
@@ -1,13 +1,18 @@
 """Config flow for Tradfri."""
+from __future__ import annotations
+
 import asyncio
+from typing import Any
 from uuid import uuid4
 
 import async_timeout
 from pytradfri import Gateway, RequestError
 from pytradfri.api.aiocoap_api import APIFactory
 import voluptuous as vol
 
-from homeassistant import config_entries
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.typing import DiscoveryInfoType
```

Comment:
That should be cleaner.
```suggestion
from homeassistant import config_entries
from homeassistant.core import HomeAssistant
from homeassistant.data_entry_flow import FlowResult
from homeassistant.helpers.typing import DiscoveryInfoType
```